Runtime/Client/LootLockerServerApi.cs

```diff
@@ -452,7 +452,7 @@ private UnityWebRequest CreateWebRequest(string url, LootLockerServerRequest req
                         webRequest = new UnityWebRequest();
                         webRequest.SetRequestHeader("Content-Type", "multipart/form-data; boundary=--");
                         webRequest.uri = new Uri(url);
-                        LootLockerLogger.Log(url);//the url is wrong in some cas, LootLockerLogger.LogLevel.Verbosees
+                        LootLockerLogger.Log(url); // The url is wrong in some cases
```

Comment:
Do we want this to stick around?

Reply:
I vote no. Removing it.
```suggestion
                        //LootLockerLogger.Log(url); // The url is wrong in some cases
```